### PR TITLE
highlight current line

### PR DESCRIPTION
**Pull request by alan-w-255**
add highligh config for task under current cursor.

### CODE REVIEW

**Comment by alan-w-255** [comment]
Cheese_Sat-06Nov21_12 14

File: src/config.rs [hunk 1]
```diff
@@ -98,13 +99,15 @@ impl Config {
         let uda_selection_dim = Self::get_uda_selection_dim(data);
         let uda_selection_blink = Self::get_uda_selection_blink(data);
         let uda_calendar_months_per_row = Self::get_uda_months_per_row(data);
+        let uda_style_selection = Self::get_uda_style("selection", data);
```
Comment (kdheepak) [inline src/config.rs:102]: Can you change this to 

```suggestion
        let uda_report_style_selection = Self::get_uda_style("report.selection", data);
```

Comment (alan-w-255) [reply src/config.rs:102]: ok

**Comment by kdheepak** [comment]
Thanks for the PR! This looks good! If you can update the naming of the attribute, I'll merge it.

**Comment by alan-w-255** [comment]
You are welcome.

**Comment by kdheepak** [comment]
I changed this behavior slightly in a new PR. If users don't define a selection, it'll now use the current task's style. If users do define a selection style, it'll patch the current task's style with the selection style.

**Comment by kdheepak** [comment]
The selection highlight behavior has been changed in this version: https://github.com/kdheepak/taskwarrior-tui/releases/tag/v0.14.10

**Pull request merged**